package.json

```diff
@@ -1,36 +1,37 @@
 {
   "name": "business-edit-ui",
-  "version": "4.5.6",
+  "version": "4.6.0",
   "private": true,
   "appName": "Edit UI",
   "sbcName": "SBC Common Components",
   "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint",
-    "lint:nofix": "vue-cli-service lint --no-fix",
-    "test:unit": "vue-cli-service test:unit --testPathPattern --coverage",
-    "test:debug": "node --inspect-brk node_modules/.bin/vue-cli-service test:unit --testPathPattern --no-cache --watch --runInBand"
+    "dev": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
```

Comment:
port 8080? maybe not call it serve? I called it `preview`

Reply:
Makes a lot sense. I was just following the official guide in this and the other PR: https://vueschool.io/articles/vuejs-tutorials/how-to-migrate-from-vue-cli-to-vite/
![migration guide](https://github.com/bcgov/business-edit-ui/assets/122301442/648ebc03-40a3-4aa9-b08a-61f5513499fa)

In the guide, they mention that `npm run serve` is `vite preview`.

I do agree with you though Travis.

Reply:
Yeah, I was bitten by this last week. I have to condition myself to type "npm run dev" now, that's all.

I like that Karim is following the official guide.